test/e2e-png.js

```diff
@@ -11,12 +11,16 @@ const fs             = require("fs");
 const fsP            = pify(fs);
 const sourceFile     = `${__dirname}/fixture/events/s3_put_file.json`;
 const setting        = JSON.parse(fs.readFileSync(sourceFile));
+const AWS            = require("aws-sdk-mock");
 
 let processor;
 let images;
 let fileSystem;
 
 test.before(async t => {
+    AWS.mock("S3", "deleteObject", (params, callback) => {
```

Comment:
Keeping in mind that you are using `sinon.spy` in line [here](https://github.com/ysugimoto/aws-lambda-image/blob/63d3bd86f454ae8fa5745c6cacc813ea3bde78a4/test/e2e-png.js#L52) is it necessary to use `AWS.mock`?